audit-service/src/main/java/uk/gov/gchq/palisade/audit/service/AuditApplication.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package uk.gov.gchq.palisade.service.audit;
```

Comment:
can we revert this change so it matches the same package format of the Palisade service. <base (uk.gov.gchq.palisade)><repo (service)><service(audit)>